lua/nvim-treesitter/parsers.lua

```diff
@@ -633,11 +633,11 @@ list.bibtex = {
 
 list.zig = {
   install_info = {
-    url = "https://github.com/Himujjal/tree-sitter-zig",
+    url = "https://github.com/maxxnino/tree-sitter-zig",
     files = { "src/parser.c" },
```

Comment:
You need to add `branch="main"`

Reply:
sorry, i don't know where to add it

Reply:
you need to add it on the `install_info` table, you can search for `branh` on this same file for reference.